Generals/Code/GameEngine/Include/Common/ScopedMutex.h

```diff
@@ -34,7 +34,10 @@ class ScopedMutex
 	public:
 		ScopedMutex(HANDLE mutex) : m_mutex(mutex)
 		{
-			WaitForSingleObject(m_mutex, INFINITE);
+			DWORD status = WaitForSingleObject(m_mutex, 500);
```

Comment:
This is a very strange change. I suggest we revert this change for this particular pull request, because it changes behaviour. I created a report with #805

Reply:
I took the liberty and reverted this one for now.